client/features/auth-pages/SignUpPage.tsx

```diff
@@ -217,12 +242,12 @@ const SignUpPage = () => {
           </Typography>
         </Box>
       </Box>
-      <SnackBarNotification
+      {/* <SnackBarNotification
```

Comment:
Any particular reason to move the snackbar inside the box component?

Reply:
No, I just moved it back. I initially wanted to see how it would react when I moved it.